docs/pages/refactoring.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Refactoring tools
+layout: default
+---
+
+# Refactoring Tools
```

Comment:
Could add sourcery.ai, free for open source. Used quite a bit, although it's got more clutter now than it used to

Reply:
Since it's a thing we've used and like, I think it should go in.